package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codexteam/icons",
-  "version": "0.1.1-rc.0",
+  "version": "0.1.2",
```

Comment:
let's use 0.2.0 version as new icons is a feature?

we need to apply the agreement for a versioning. we had a discussion with @neSpecc about this project.

`*.*.X` — patch for fixes 
`*.X.0` — minor for features 
`X.0.0` — major for no backwards compatibility features

Reply:
okay, updated the version